crates/ruff/src/rules/pydocstyle/rules/capitalized.rs

```diff
@@ -40,36 +40,27 @@ pub fn capitalized(checker: &mut Checker, docstring: &Docstring) {
     }
 
     let body = docstring.body();
-
     let Some(first_word) = body.split(' ').next() else {
         return
     };
-    if first_word == first_word.to_uppercase() {
-        return;
-    }
```

Comment:
I think our `uppercase_first_char` check will always be true here anyway, so this seems unnecessary? And it requires an allocation.